sql/5_0_2-to-6_0_0_upgrade.sql

```diff
@@ -461,8 +461,8 @@ INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`
 ALTER TABLE `form_eye_mag_prefs` MODIFY `ordering` smallint(6) DEFAULT NULL;
 #EndIf
 
-#IfNotColumnType codes code_text_short varchar(255)
-ALTER TABLE `codes` MODIFY `code_text_short` varchar(255) NOT NULL default '';
+#IfNotColumnType codes code_text_short text
+ALTER TABLE `codes` MODIFY `code_text_short` text NOT NULL default '';
```

Comment:
the nice thing about text is it just gets to be by itself:
```
ALTER TABLE `codes` MODIFY `code_text_short` text;
```
(would do the same for the other places that are changed to text in this pr)